react-common/components/controls/Feedback/configs.ts

```diff
@@ -0,0 +1,57 @@
+export const appId = 50315;
+export const feedbackFrameUrl = 'https://admin-ignite.microsoft.com';
+
+export const baseConfig: any = {
+    feedbackUiType: "NoSurface",
+    hostPlatform: "IFrame",
+    isDisplayed: true,
+    isEmailCollectionEnabled: false, // to enable email collection
+    isFileUploadEnabled: false, // to enable file upload function
+    isScreenshotEnabled: false, // to enable screenshot
+    isScreenRecordingEnabled: false, // to enable screen recording
+    invokeOnDismissOnEsc: false,
+    isFeedbackForumEnabled: false,
+    isMyFeedbackEnabled: false,
+    isThankYouPageDisabled: false,
+}
+
+export const ratingFeedbackConfig: any = {
+    ...baseConfig,
+    initialFeedbackType: "Unclassified",
+    scenarioConfig: {
+      isScenarioEnabled: true,
+      scenarioType: "Custom",
+      questionDetails: {
+        questionUiType: "Rating",
+        questionInstruction: {
+          displayedStringInEnglish: "What did you think of this activity?",
+          displayedString: "What did you think of this activity?"
+        },
+        questionOptions: [
+          {
+            displayedStringInEnglish: "Boring",
+            displayedString: "Boring"
+          },
+          {
+            displayedStringInEnglish: "Not fun",
+            displayedString: "Not fun"
+          },
+          {
+            displayedStringInEnglish: "Kinda fun",
```

Comment:
nit: I'd recommend "kind of fun" instead of "kinda fun"

Reply:
Mostly because I wonder if "kinda" will bother picky teachers

Reply:
Yeah, definitely. I'm not expecting we keep these. These are basically placeholders as I haven't gotten PM feedback for this yet. The rating one really isn't being used yet, either, so I would like to save this change when I get the rating feedback working if that's alright.